cmd/server/go.mod

```diff
@@ -0,0 +1,153 @@
+module github.com/uber/cadence/cmd/server
+
+go 1.20
+
+replace github.com/uber/cadence => ../../
```

Comment:
Maybe we can add a one liner for folks to understand the problem being solved here - if that's easy enough to concretize in a line or two?

Reply:
yea... lemme see if I can come up with anything.
there probably is something small.  it's pretty straightforward from what replaces _do_, but that's far from widely known.